zerver/tests/test_events.py

```diff
@@ -911,6 +911,7 @@ def test_custom_profile_fields_events(self) -> None:
         schema_checker = self.check_events_dict([
             ('type', equals('custom_profile_fields')),
             ('fields', check_list(check_dict_only([
+                ('id', check_int),
```

Comment:
The reason it wasn't throwing error was, `fields` was always `[]`. So schema checker for fields wasn't get tested.